docker/Dockerfile.deploy

```
@@ -11,9 +11,14 @@ ARG PORT=4321
 ENV PORT=${PORT}
 
 WORKDIR /usr/src/app
+<<<<<<< HEAD
```

Hadolint (2.14.0)

[error] unexpected '<'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)